# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/fox/hotfix-libjpeg.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- fox-1.7.85/lib/fxjpegio.cpp.vanilla	2025-01-22 16:57:34.593294300 +0100
+++ fox-1.7.85/lib/fxjpegio.cpp	2025-01-22 16:58:32.656628749 +0100
@@ -373,7 +373,7 @@
   dstinfo.dest=&dst.pub;
 
   // Based on Sander's recommendation
-  dstinfo.optimize_coding=true;
+  dstinfo.optimize_coding=(boolean)true;
 
   jpeg_set_defaults(&dstinfo);
   jpeg_set_quality(&dstinfo,quality,TRUE);
